src/app.js

```diff
@@ -59,7 +59,6 @@ function listen (koaApp, config, ledgers) {
   // Start a coroutine that connects to the backend and
   // subscribes to all the ledgers in the background
   co(function * () {
-    yield ledgers.checkLedgersHealth()
```

Comment:
We are now checking ledger health via the websocket status. Websocket do regular pings automatically to ensure the connection is still live, so it's a more reliable mechanism than checking the health endpoint once at startup.